build/prbuild.yml

```diff
@@ -95,3 +98,20 @@ jobs:
       platform: '$(BuildPlatform)'
       configuration: debug
       rerunFailedTests: true
+
+- job: build_without_fakes
+  pool:
+    vmImage: 'vs2017-win2016'
+  steps:
+  - task: NuGetToolInstaller@0
+    displayName: 'Use NuGet 4.3.0'
+
+  - task: NuGetCommand@2
+    displayName: 'NuGet restore'
+
+  - task: VSBuild@1
+    displayName: 'Build Solution **\*.sln'
+    inputs:
+      vsVersion: 15.0
+      platform: '$(BuildPlatform)'
+      configuration: debug
```

Comment:
>debug [](start = 21, length = 5)

What would happen if both the debug and the build_without_fakes jobs were to run on the same agent instance? Is there a guarantee that the build folders will be isolated between the agents?

Reply:
I believe so. If you go to any of the build pipelines, click on 'edit pipeline' (in the dropdown with '...' as its label on the top right), and then on 'get sources', you can see 'clean' is set to true.